proto/cosmos/tx/v1beta1/tx.proto

```diff
@@ -63,6 +63,33 @@ message SignDoc {
   uint64 account_number = 4;
 }
 
+// SignDocTextual is the type used for generating sign bytes for SIGN_MODE_TEXTUAL. It is designed to be serialized
+// as proto3 JSON following the rules defined Here: https://github.com/regen-network/canonical-proto3/blob/master/README.md#json.
+message SignDocTextual {
+  // body is the processable content of the transaction
+  TxBody body = 1;
+
+  // auth_info is the authorization related content of the transaction,
+  // specifically signers, signer modes and fee
+  AuthInfo auth_info = 2;
+
+  // chain_id is the unique identifier of the chain this transaction targets.
```

Comment:
```suggestion
  // chain_id is the identifier of the chain this transaction targets.
```

Without a registry we can't ensure uniqueness for those self-chosen identifiers. This is probably a different discussion, but I'd just avoid any potentially misleading properties here.